docs-src/traffic-shifting.md

```diff
@@ -0,0 +1,97 @@
+# HTTP traffic shifting
```

Comment:
Tiny Nit: Is "Traffic Splitting" or "Traffic Shifting" a more common term? It looks like both terms are used throughout the doc, just trying to think of which users will be more likely to search for.

Reply:
Just saw that you asked about this in the PR as well. Between traffic shifting, traffic splitting, and traffic weighting, I have a slight preference towards traffic splitting, followed by traffic shifting. Interested in what others think as well.

Reply:
I like traffic splitting over shifting.

Reply:
traffic splitting wins then! I'm switching over everything for traffic splitting.